### DISCUSSION TITLE
solid with parcel?

### DISCUSSION THREAD

**Discussion by ghost** [solidjs/solid#837] [Q&A] [answered]
Since vite [a little uncomfortable](https://github.com/vitejs/vite/issues/257) in multipage. Is there an example of integration solid with parcel? Or some other similar tool

**Comment by atk** [comment, marked as answer]
Actually, solid-start with pre-rendered routes (that still uses vite) is pretty awesome. But anyways, here is a starter template for parcel: https://github.com/LXSMNSYC/solid-parcel-template

**Comment by ghost** [comment]
@atk Oh thanks. Didn't know about solid-start
